### ISSUE TITLE
Regression: missing accept4(2) in 0.2.87 for arm-linux-androideabi target

### ISSUE THREAD

**Issue by Thomasdezeeuw** [rust-lang/libc#2101]
Missing accept4(2) in `arm-linux-androideabi` target in libc 0.2.87.

To reproduce:

```toml
[package]
name = "test"
version = "0.1.0"
authors = ["Thomas de Zeeuw <thomasdezeeuw@gmail.com>"]
edition = "2018"

[dependencies]
libc = "=0.2.86" # Change to to .87 to fail.
```

```rust
fn main() {
    unsafe { libc::accept4(0, std::ptr::null_mut(), std::ptr::null_mut(), 0) };
}
```

Related: https://github.com/tokio-rs/mio/issues/1467.

**Comment by JohnTitor** [comment]
Should be caused by #2079, cc @de-vri-es

**Comment by de-vri-es** [comment]
Fix proposed in PR #2102
